src/gawronski-bartosz/Fsd.Bartek.Cs/Fsd.Bartek.Ex4.Services/Sql/Sql.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Bartek.Ex4.Services.Sql
+{
+    public static class Sql
```

Comment:
nigdzie nie ma connection.close(). Najlepiej to zrobić implementując interfejs IDisposable